internal/pkg/service/buffer/store/slice_test.go

```diff
@@ -0,0 +1,87 @@
+package store
+
+import (
+	"context"
+	"testing"
+
+	"github.com/benbjohnson/clock"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
+)
+
+func TestStore_CreateSlice(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	store := newStoreForTest(t)
+
+	now := clock.NewMock().Now().UTC()
+	receiverKey := key.ReceiverKey{ProjectID: 1000, ReceiverID: "github"}
+	exportKey := key.ExportKey{ExportID: "github-issues", ReceiverKey: receiverKey}
+	fileKey := key.FileKey{FileID: now, ExportKey: exportKey}
+	sliceKey := key.SliceKey{SliceID: now, FileKey: fileKey}
+	slice := model.Slice{
+		SliceKey:    sliceKey,
+		SliceNumber: 1,
+	}
+	_, err := store.createSliceOp(ctx, slice).Do(ctx, store.client)
+	assert.NoError(t, err)
+
+	// Check keys
+	etcdhelper.AssertKVs(t, store.client, `
+<<<<<
+slice/1000/github/github-issues/1970-01-01T00:00:00.000Z/1970-01-01T00:00:00.000Z
+-----
+{
+  "projectId": 1000,
+  "receiverId": "github",
+  "exportId": "github-issues",
+  "fileId": "1970-01-01T00:00:00Z",
+  "sliceId": "1970-01-01T00:00:00Z",
```

Comment:
Tieto hodnoty by mohli byt vyplnene, ked testujeme prave tento objekt.
Co je ale dolezitejsie, hovori mi to, ze nam ta validacia nejak nefunguje nie? Nie je tato hodnota `IsZero`? Nemal sa vyhodit `panic`?

Reply:
Teraz na to pozeram druhy krat a vidim tam `now := clock.NewMock().Now().UTC()`, OK.

Uplne by stacilo `time.Parse` nejakej stacitkej hodnoty, hlavne nech:
- to nie je ten zero time
- a tie hodnoty su rozne

Reply:
Hm, zkusím. Ten Parse jsem tam chvíli měl, ale docela dlouho jsem včera bojoval s tím že nějak neseděla time zóna. Až po přidání UTC() to začalo fungovat.

Reply:
Mozes to skopirovat z RecordKey testu, tam sa to parsuje aj s timezone, nejake `2006....`.
`Utc()` riesi `FormatTime`, takze ako to porovnas ako staticky string, tak by tam mal byt ten isty datum ale v UTC.

Reply:
Mně se ty porovnávaný timestampy pořad lišily myslím o osm hodin. Ten mockovanej čas je v nějaký "local" zóně nebo co, jestli si dobře pamatuju. Podívám se, díky